src/Library/demos/Event Controllers/main.blp

```diff
@@ -0,0 +1,116 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage page {
+  title: _("Event Controllers");
+  description: _("Ancillary Objects which responds to events");
+
+  child: Box {
```

Comment:
Instead of using the `margin-*` properties on all the children, you should just use the `spacing` property on this parent box.